src/app/notification.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+import { NotificationMessage, NotificationType } from './notification.message';
+import { ToastrService } from 'ngx-toastr';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class NotificationService {
+
+  private notificationSubject: Subject<NotificationMessage> = new Subject<NotificationMessage>();
+  
+  sendMessage(message: NotificationMessage) {
+    this.notificationSubject.next(message);
+  }
+
+  constructor(private toastrService: ToastrService) {
+    this.notificationSubject.subscribe(message => {
+      switch (message.type) {
+        case NotificationType.success:
+          this.toastrService.success(message.message);
+          break;
+        case NotificationType.error:
+          this.toastrService.error(message.message);
+          break;
+        case NotificationType.warning:
+          this.toastrService.warning(message.message);
+          break;
+        case NotificationType.info:
```

Comment:
dwa razy mamy NotificationType.info. Chyba da się razem w switchu zapisać default i NotificationType.info

Reply:
Tutaj robiłem za instrukcją, ale zobaczę, czy przyjmie to jako jedną opcję